src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -46,6 +46,8 @@
 
     private double slackBusPMaxMismatch = SLACK_BUS_P_MAX_MISMATCH_DEFAULT_VALUE;
 
+    private boolean computeMainConnectedComponentOnly = COMPUTE_MAIN_CONNECTED_COMPONENT_ONLY_DEFAULT_VALUE;
```

Comment:
Now that you have switched to RC, you don't need to add this specific parameter as it is in the base parameters?